src/components/add-button/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./style.css";
+import { Plus } from "phosphor-react";
+
+interface AddButtonProps {
+  onClick?: () => string;
+}
+
+export const AddButton: React.FC<AddButtonProps> = ({ onClick }) => {
+  return (
+    <>
+      <div className="btn-container">
+        <div className="btn-add" onClick={onClick}>
+          <div className="btn-text">Adicionar Usuário</div>
+          <Plus className="icon" size={24} weight="bold" />
+        </div>
+      </div>
+    </>
```

Comment:
não precisa do react.fragment! 
```suggestion
      <div className="btn-container">
        <div className="btn-add" onClick={onClick}>
          <div className="btn-text">Adicionar Usuário</div>
          <Plus className="icon" size={24} weight="bold" />
        </div>
      </div>
```

Reply:
Realmente, o PR ficou muito grande. Fui concertar algumas coisas e acabei me empolgando, daí quando fui ver já era 'tarde demais' hahaha

Mas obrigado Beto, vou me atentar mais a isso!